app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/CompleteClaim.scala

```diff
@@ -50,7 +50,8 @@ final case class CompleteClaim(
   claimsAnswer: ClaimsAnswer,
   reimbursementMethodAnswer: Option[ReimbursementMethodAnswer],
   scheduledDocumentAnswer: Option[ScheduledDocumentAnswer],
-  associatedMRNsAnswer: Option[AssociatedMRNsAnswer]
+  associatedMRNsAnswer: Option[AssociatedMRNsAnswer],
+  typeOfClaim: Option[SelectNumberOfClaimsAnswer]
```

Comment:
It is probably a good opportunity to rename `SelectNumberOfClaimsAnswer` to `ClaimType` or `TypeOfClaim`, wdyt?

Reply:
updated